File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);
    int cents = round(dollars * 100);
    int coins = cents / 25;
    cents = cents % 25;
    coins = coins + cents / 10;
    cents = cents % 10;
    coins = coins + cents / 5;
    cents = cents % 5;
    coins = coins + cents / 1;
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

dollars = -1
while dollars < 0:
    dollars = get_float("Change owed: ")
cent = round(dollars * 100)

n = cent // 25
cent %= 25
n = n + cent // 10
cent %= 10
n = n + cent // 5
cent %= 5
n = n + cent // 1

print(n)